cert-data.yml

```diff
@@ -1,4 +1,179 @@
 ---
+### 4_programmatic Default course
+Default/OpenEdX/Certificate: &cert_defaults
+  VERSION: 4_programmatic
+  TEMPLATEFILE: False
+  instructors:
+    - instructor: &instructor_1
+        x_position: 475
+        y_position: 140
+    - instructor: &instructor_2
+        x_position: 475
+        y_position: 220
+    - instructor: &instructor_3
+        x_position: 475
+        y_position: 300
+    - instructor: &instructor_4
+        x_position: 475
+        y_position: 380
+    - instructor: &instructor_5
+        x_position: 275
+        y_position: 140
+    - instructor: &instructor_6
+        x_position: 75
+        y_position: 140
+  instructor_template:
+    # ordered list of steps to render the instructor signature block, bottom to top
+    - text:
+        string: "school"
```

Comment:
Especially since we actually do have interpolated strings throughout this file, I would prefer we use single quotes for all non-interpolated strings.